server.js

```diff
@@ -170,6 +174,40 @@ async function createExpressApp() {
       })
   })
 
+
+expressApp.post('/createRoom', validateCreateRoomParams, async (req, res) => {
+  const { eventNotificationUri, maxPeersRoom, roomId } = req.body;
+
+  // If roomId is not provided, generate a random one
+  const roomIdToUse = roomId || randomString({ length: 8, numeric: true, letters: true }).toLowerCase();
+  // Port of connection websocket
+  const port = process.env.MEDIASOUP_CLIENT_PROTOOPORT
+  // Host or Ip where try connect ws
+  const announcedIp = config.https.ingressHost
+  // Ws url to response
+  const wsUrl = `wss://${announcedIp}:${port}?roomId=${roomIdToUse}&peerId=`
```

Comment:
```suggestion
  const wsUrl = `wss://${announcedIp}:${port}?roomId=${roomIdToUse}`
```

I think it would be better to leave the client add the whole key/value

Reply:
fixed!